new-resource.md

```diff
@@ -0,0 +1,92 @@
+---
+layout: default
+title: New Resource
+stylesheet: /new-resource.css
+---
+
+<a href="/">Back to the list</a>
+
+<h2>New Resource Request</h2>
+
+<p>
+    Thank you for taking an interest in helping us expand this list of resources. To request a new resource be added, please fill the form below. Please note that the information sent in the form is not private and can be read by our web host, Netlify. There's nothing we can do about this at this time without significantly increasing the cost to run this website. If you'd prefer to contact us directly, please email us at <a href="mailto:techblocsea@protonmail.com">techblocsea@protonmail.com</a>.
+</p>
+
+<!-- This form won't work locally but netlify transforms the `action` into a "success" page that it redirects to after handling the POST request -->
+<form name="new-resource" data-netlify="true" action="/new-resource-thanks" method="POST">
+    <p>An asterisk (*) denotes a required field</p>
+    <p>
+        <label for="submitter-email">Your Email (optional):</label>
+        <aside>Include this if you feel comfortable with us following up with you about this resource. It is extremely helpful for us to be able to do so and will increase the likelihood that the resource will be included.</aside>
+        <input id="submitter-email" type="email" name="submitter-email" />
+    </p>
+    <p>
+        <label class="required">Resource name: <input type="text" required name="name" /></label>
+    </p>
+    <p>
+        <!-- Don't use type="url" here because it's too strict and requires a schema which some users might not bother to enter and could be confusing -->
+        <label>Resource URL (if available): <input type="text" name="url" /></label>
+    </p>
+    <p>
+        <label class="required" for="phone">Resource phone:</label>
+        <aside>If there are additional phone numbers, please list them in the dscription</aside>
+        <input id="phone" type="tel" required name="phone" />
+    </p>
+    <p>
+        <label for="resource-email">Resource email:</label>
+        <aside>If available, any contact you have at the organization behind the resource or a general purpose email for the organization</aside>
+        <input id="resource-email" type="email" name="resource-email" />
+    </p>
+    <p>
+        <label class="required" for="category">Resource category:</label>
+        <aside>Please choose at least one. To select multiple on PC, hold control (or command if on a mac) and click on each option you wish to select.</aside>
```

Comment:
"Please choose at least one. To select multiple, hold Control (on PC), or Command (On Mac), and click on each option you wish to select."

Reply:
Done in https://github.com/OrcaCollective/not911.me/pull/52/commits/0def96f7f212b6e3c6312226e029a9f1b7478854